recipes/proj/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1.3)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.c)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

Comment:
Locally, I had to add
```
set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
```

This is because conan does not propagate the required linker language.
@uilianries Is there a solution to this?

Reply:
if source code is in C++, just use .cpp extension